lms/djangoapps/discussion/static/discussion/templates/fake-breadcrumbs.underscore

```diff
@@ -1,6 +1,8 @@
 <h6 class="hd-6 breadcrumbs">
     <span class="nav-item">
-        <a class="all-topics" href="">All Topics</a>
+        <a class="all-topics" href="">
+            <span class="icon fa fa-bars" aria-hidden="true"></span>All Topics
```

Comment:
I possibly missed in a previous iteration, but I think this needs to be marked for translation.